### PR TITLE
[ASV-355] AppCoinsAds bundle

### PR DESCRIPTION
**Pull request by phillipsgoncalves**
**What does this PR do?**

 Shows AppCoins ads on the Get Rewarded with AppCoins bundle. At the moment there are no campaigns available, so you must rewrite the request on charles to get apps. Also, if you navigate to the AppView of a rewarded App you will see a card showing how many appcoins you will get for trying that app.

 
**Database changed?**

    No

**Where should the reviewer start?**

- [ ] BundlesResponseMapper.java

**How should this be manually tested?**
Open app and check for the Get Rewarded with AppCoins Bundle

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-355](https://aptoide.atlassian.net/browse/ASV-355)

**Questions:**

No.

**Code Review Checklist**

- [ ] Architecture
- [ ] Documentation on public interfaces
- [ ] Database changed?
- [ ] If yes - Database migration?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] Partners build
- [ ] Unit tests pass
- [ ] Functional QA tests pass

### CODE REVIEW

**Comment by analaragomes** [comment]
Why we write: 1,11 APPC in the preview card and 1,121321324143 APPC in the app view?

**Comment by analaragomes** [comment]
Bug: the new card "Try this app and get ... APPC" is always showed in app view. Should only be showed in apps with asf ads flow

File: app/src/main/java/cm/aptoide/pt/app/view/widget/AppViewRewardAppWidget.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  }
+
+  @Override public void unbindView() {
+    super.unbindView();
+    displayable = null;
+    twoDecimalFormat = null;
+  }
+
+  @Override public void bindView(AppViewRewardAppDisplayable displayable) {
+    this.displayable = displayable;
```
Comment (JDSM01) [inline app/src/main/java/cm/aptoide/pt/app/view/widget/AppViewRewardAppWidget.java]: Code Smell: not needed code thinking on future needs

File: app/src/vanilla/java/cm/aptoide/pt/app/view/AppViewFragment.java [hunk 2]
```diff
@@ -316,10 +332,10 @@ public void setAppName(String appName) {
   }
 
   @Override public void onCreate(@Nullable Bundle savedInstanceState) {
+    appRewardAppcoins = 0;
```
Comment (JDSM01) [inline app/src/vanilla/java/cm/aptoide/pt/app/view/AppViewFragment.java]: It shouldn't be needed here, but maybe test it a bit, just to be sure

**Pull request merged**